geometry_msgs/Point[] statei